distribution/lib/Standard/Base/0.0.0-dev/src/IO/Prim_Io.enso

```diff
@@ -0,0 +1,29 @@
+## Primitive IO operations internal to the runtime.
+type Prim_Io
+
+    ## PRIVATE
+
+       A type for primitive IO operations.
+    type Prim_Io
+
+## PRIVATE
+
+   Gets a file corresponding to the current working directory of the
+   program.
+get_cwd : File
+get_cwd = @Builtin_Method "Prim_Io.get_cwd"
```

Comment:
A different solution here – all of these should be private methods inside `File.enso` i.e. next the only file that uses them, the `Prim_Io` is pointless now.

Reply:
This one will be a bit more involving since to get the actual function node we lookup based on the owner that will no longer be `Prim_Io`. But we can change the type param for @BuiltinMethod for those to `File`.

Reply:
done